prod/westeurope/internal/api/functions_admin_r3/function_app/terragrunt.hcl

```diff
@@ -0,0 +1,154 @@
+}
+
+dependency "storage_container_user-data-download" {
+  config_path = "../../storage_user-data-download/container_user-data-download"
+}
+
+dependency "storage_account_userbackups" {
+  config_path = "../../storage_userbackups/account"
+}
+
+dependency "storage_container_user-data-backup" {
+  config_path = "../../storage_userbackups/container_user-data-backup"
+}
+
+dependency "app_service_appbackend" {
+  config_path = "../../../appbackend/app_service"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v2.0.34"
+}
+
+inputs = {
+  name                = "admin"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+
+  application_insights_instrumentation_key = dependency.application_insights.outputs.instrumentation_key
+
+  resources_prefix = {
+    function_app     = "fn3"
+    app_service_plan = "fn3"
+    storage_account  = "fn3"
+  }
+
+
+  pre_warmed_instance_count = 1
+
+  runtime_version = "~3"
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME     = "node"
+    WEBSITE_NODE_DEFAULT_VERSION = "10.14.1"
+    WEBSITE_RUN_FROM_PACKAGE     = "1"
+    NODE_ENV                     = "production"
+
+    COSMOSDB_URI               = dependency.cosmosdb_account.outputs.endpoint
+    COSMOSDB_KEY               = dependency.cosmosdb_account.outputs.primary_master_key
+    COSMOSDB_NAME              = dependency.cosmosdb_database.outputs.name
+    COSMOSDB_CONNECTION_STRING = dependency.cosmosdb_account.outputs.connection_strings[0]
+
+    StorageConnection = dependency.storage_account.outputs.primary_connection_string
+
+    AssetsStorageConnection = dependency.storage_account_assets.outputs.primary_connection_string
+
+    AZURE_APIM                = "io-p-apim-api"
+    AZURE_APIM_HOST           = "api-internal.io.italia.it"
+    AZURE_APIM_RESOURCE_GROUP = "io-p-rg-internal"
+
+    MESSAGE_CONTAINER_NAME = dependency.storage_container_message-content.outputs.name
+
+    UserDataArchiveStorageConnection = dependency.storage_account_user-data-download.outputs.primary_connection_string
+    USER_DATA_CONTAINER_NAME         = dependency.storage_container_user-data-download.outputs.name
+
+    PUBLIC_API_URL           = "http://api-internal.io.italia.it/"
+    PUBLIC_DOWNLOAD_BASE_URL = "https://${dependency.storage_account_user-data-download.outputs.primary_blob_host}/${dependency.storage_container_user-data-download.outputs.name}"
+
+    SESSION_API_URL                 = "https://${dependency.app_service_appbackend.outputs.default_site_hostname}"
+    UserDataBackupStorageConnection = dependency.storage_account_userbackups.outputs.primary_connection_string
+    USER_DATA_BACKUP_CONTAINER_NAME = dependency.storage_container_user-data-backup.outputs.name
+    USER_DATA_DELETE_DELAY_DAYS     = 7
+
```

Comment:
SLOT_TASK_HUBNAME = "ProductionTaskHub"?

Reply:
we're going to sett USER_DATA_DELETE_DELAY_DAYS     = 6 (not 7) for technical reason